src/node_api_types.h

```diff
@@ -67,6 +67,7 @@ typedef enum {
   napi_generic_failure,
   napi_pending_exception,
   napi_cancelled,
+  napi_escape_called_twice,
```

Comment:
It is public.  At this point since we are experimental it may be ok, but more generally we have to be able to add new error codes so assuming adding to an enum cannot preserve the abi we will have to define our errors in a different way.